results/15_RandomForest_LatentVar_TumorType.Rmd

```diff
@@ -0,0 +1,280 @@
+---
+title: "RF_latent_variable"
+author: "Jineta Banerjee"
+date: '`r format(Sys.time(), "%d %B, %Y")`'
+output: 
+  html_document:
+    toc: true
+    number_sections: true
+    toc_float:
+      collapsed: false
+      smooth_scroll: false
+    fig_width: 7
+    fig_height: 6
+    fig_caption: true
+    df_print: paged
+    code_folding: hide
+    
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
```

Comment:
this keeps all of the code from showing up (not even folded), it might be nice to include the code chunks but leave folding on?

Reply:
fixed